frontend/src/Dashboard/CrossingListPage/CrossingListHeader/CrossingListHeader.scss

```diff
@@ -0,0 +1,104 @@
+.CrossingListHeader.fullsize {
+  height: 65px;
+  background: #0C0333;
+  display: flex;
+}
+
+.CrossingListHeader.smallsize {
+  background: #0C0333;
+  min-width: 240px;
+}
+
+.CrossingListSearch.fullsize {
+  margin-left: 5%;
+  flex: 1;
+  display: flex;
+  min-width: 200px;
+}
+
+.CrossingListSearch.smallsize {
+  display: flex;
+  margin-left: 5%;
+  margin-right: 5%
+}
+
+.CrossingListSearchInput {
+  height: 45px;
+  box-sizing: border-box;
+  width: 85%;
+  margin-top: 10px;
+  font-size: 1.4em;
+  padding-left: 1em;
+  min-width: 155px;
+}
+
+.CrossingListSearchButton {
+  background: #1764F6;
+  width: 15%;
+  min-width: 30px;
+  height: 45px;
+  margin-top: 10px;
+  color: white;
+  font-size: 1.6em;
+  text-align: center;
+  line-height: 45px;
+}
+
+.CrossingListSortToggle.fullsize {
+  flex: auto;
+  color: white;
```

Comment:
There are examples of this scattered throughout, but maybe another place for global Sass variables.